configuration/shared_subscriptions.md

```diff
@@ -22,3 +22,23 @@ shared_subscription_policy = prefer_local
 
 When a messages is being delivered to subscribers of a shared subscription, the message will be delivered to an online subscriber if possible, otherwise the message will be delivered to an offline subscriber.
 
+## Examples
+
+**Subscriptions**
+*Note: When subscribing to a shared topic, make sure to escape the `$` 
+```bash
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname/#
+```
+So, for dash on ubuntu bionic
```

Comment:
Typo: bash instead of bash?

Reply:
No, Ubuntu actually used dash. https://wiki.ubuntu.com/DashAsBinSh. I always though bash as well, but doing "man sh" returns a dash manual page.

Reply:
Since it's the same for the dash and bash shells I think it would be better to write  sth like `So, for the bash and dash shells` and leave out the ubuntu bionic parts.

Reply:
I like that. I'll make the change.